index.js

```diff
@@ -67,7 +87,16 @@ function getAverageIMDBRating() {}
  *  countByRating(movies);
  *  //> { G: 3, PG: 7 }
  */
-function countByRating() {}
+function countByRating(movies) {
+  let movieCount = 0
+  let movObj = {rated: movieCount}
+
+  for (let i = 0; i < movies.length; i ++){
+  // if (movies[i].rated === 
+  // }return movObj
+}
```

Comment:
This is a tricky problem - how can we dynamically assign an object a key ?  Remember we can pass a string to an object using `[bracket notation]` to create a key.  When do we want to create a key - and what do we need to do if there already is one?